### PR TITLE
Fixing some code errors

### PR DESCRIPTION
**Pull request by castulo**
Signed-off-by: Castulo Martinez <castulo.martinez@intel.com>

### CODE REVIEW

File: src/search_file.c [hunk 1]
```diff
@@ -166,7 +166,10 @@ static long get_bundle_size(const char *bundle)
 		return b->size;
 	}
 
-	b = calloc(1, sizeof(struct bundle_size));
+	b = (struct bundle_size*)calloc(1, sizeof(struct bundle_size));
```
Comment (otaviobp) [inline src/search_file.c]: we don't need to cast void*, so it's fine to do:

     	b = calloc(1, sizeof(struct bundle_size));

File: src/search_file.c [hunk 2]
```diff
@@ -166,7 +166,10 @@ static long get_bundle_size(const char *bundle)
...
+	if (!b) {
+		abort();
```
Comment (otaviobp) [inline src/search_file.c]: oh, It's better not to call abort here. Use the ON_NULL_ABORT macro

Comment (castulo) [reply src/search_file.c]: ahh good call

File: src/search_file.c (hunk 1 above)
Comment (castulo) [reply src/search_file.c]: nice, I was unsure

**Pull request merged**